doc/nrf/app_dev/device_guides/nrf54l/kmu_basics.rst

```diff
@@ -0,0 +1,60 @@
+.. _ug_nrf54l_developing_basics_kmu:
+
+Introducion to KMU key provisioning
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54L devices are equipped with a Key Management Unit (KMU) that facilitates secure and confidential storage of keys.
+This feature is crucial not only for private keys but also for public keys, as the :ref:`KMU can directly transfer a key to the CRACEN RAM<ug_nrf54l_crypto_kmu_cracen_peripherals>`.
+Even when keys must pass through addressable RAM, the KMU significantly reduces the risk of key exposure.
+Therefore, you should use KMU for managing secrets whenever possible.
+
+key types
+*********
+
+Revocable keys
+==============
+
+Keys can be set as invalid for further use. The revocation policy (RPOLICY) must marked as ``revoked``for such key.
+For some technical solutions, a few keys of the same key type (for example, generation 0, 1, 2) might be provisioned.
+At any given time, only one generation should be active.
+If a key generation is compromised, it should be revoked to prevent its use.
+The specifics of the revocation scheme depend on the technical solution, but it is recommended to designate one generation as non-revocable (``locked``) to prevent a Denial of Service (DoS) attack on the key's availability for the solution.
+
+Locked keys
+===========
+
+Locked keys are permanently available for use, with an RPOLICY of ``locked``.
+
+Provisioning keys for the bootloader
+************************************
+
+The bootloader can use multiple key generations for image verification (up to three for nRF54L SoCs).
```

Comment:
Consider adding a note informing that bootloader uses nRF Security through PSA crypto APIs to access HW crypto features

Reply:
I think that unwanted info there - that's true but isn't necessary for understanding the content, ant it's a implementation detail.

Reply:
We already added the:
```
In PSA abstraction key types are mapped by ``psa_set_key_lifetime attribute``, refer to :ref:`PSA Key programming model<ug_nrf54l_crypto_kmu_key_programming_model>`.
```
Should be good enough :)